homeassistant/components/switch/rainmachine.py

```diff
@@ -42,10 +43,12 @@
             vol.Email(),  # pylint: disable=no-value-for-parameter
             vol.Required(CONF_PASSWORD):
             cv.string,
+            vol.Optional(CONF_PORT):
+            cv.port,
+            vol.Optional(CONF_SSL):
+            cv.boolean,
             vol.Optional(CONF_ZONE_RUN_TIME, default=DEFAULT_ZONE_RUN_SECONDS):
-            cv.positive_int,
-            vol.Optional(CONF_HIDE_DISABLED_ENTITIES, default=True):
```

Comment:
Got it, good to know – will keep that in mind going forward.

>Although, if there's a feature or non-feature in the code, you can usually be sure that there's someone that depends on it.

That made me LOL. 😆